postman/schemas/transfer.yml

```diff
@@ -593,6 +593,230 @@ paths:
             application/json:
               schema:
                 "$ref": "#/components/schemas/response401"
+  "/quotes":
+    get:
+      tags:
+      - Helpers
+      summary: Get Quotes
+      description: |
+        Given a corridor and a sender amount, this operation returns
+        quotes which include the exchange rate, fee adjustments, disclosures, and
+        recipient amount.<br>
+        The app can present the
+        quotes to the end-user who can decide whether to proceed with the transfer. 
+        By offering multiple quotes, end-users can compare different options for delivery speed, cost, and exchange rates, helping them make an informed decision.<br>
+        Quotes are accurate at the instance of the quote, but rates may change between
+        the time of the quote and the time of the transfer. The Execute Transfer operation
+        returns the final exchange rate.<br><br>
+        __Notes__:
+        - See [Currency](https://developer.readyremit.com/reference/about-the-reference#currency)
+        for more about currency values and decimal places.
+        - If __transferMethod__ is not supplied, quotes will be generated for the corridor's available transfer methods.
+        - The __id__ returned for a specific quote will need to be supplied to the Execute Transfer operation.  It will be assigned to __quoteHistoryId__ in the Body Params. 
+        - The __200 response__ could contain both quotes and errors if no __transferMethod__ is supplied since it may have attempted to generate more than one quote.
+
+        
+      operationId: getQuotes
+      parameters:
+      - name: destinationCountryIso3Code
+        in: query
+        description: See [country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3).
```

Comment:
The description makes sense for validation and format but we should leverage this field to describe what the field is used for. Even if the name of the field seems self-explanatory.